test_run.lua

```diff
@@ -215,7 +215,7 @@ local function get_cfg(self, name)
     return self.run_conf[name]
 end
 
-local function grep_log(self, node, what, bytes)
+local function grep_log(self, node, what, bytes, noreset)
```

Comment:
I would add `opts` parameter with `noreset` field to give a reader an idea what a parameter means in a place where the function invoked. It also allow to add more options in the future w/o large cout of parameters.

So, the code will lokk like so:

```
local function grep_log(self, node, what, bytes, opts)
    local opts = opts or {}
    local noreset = opts.noreset or false
    ...
```

Reply:
👍 **+1**